app-crypt/gpgme/gpgme-1.23.2.recipe

```diff
@@ -6,87 +6,77 @@ Currently it uses GnuPG as its backend but the API isn't restricted to \
-libqgpgmeVersion="7.3.2"
+libgpgmeVersion="11.32.0"
+libgpgmeppVersion="6.20.1"
+libqgpgmeVersion="15.5.1"
+libgpgmeVersionCompat="$libgpgmeVersion compat >= ${libgpgmeVersion%%.*}"
+libgpgmeppVersionCompat="$libgpgmeppVersion compat >= ${libgpgmeppVersion%%.*}"
+libqgpgmeVersionCompat="$libqgpgmeVersion compat >= ${libqgpgmeVersion%%.*}"
 
-#TODO: fix gpgme-config hardcoded package paths
 PROVIDES="
-	gpgme${secondaryArchSuffix} = $portVersion compat >= 1.9
-	cmd:gpgme_json${secondaryArchSuffix} = $portVersion compat >= 1.9
-	cmd:gpgme_tool${secondaryArchSuffix} = $portVersion compat >= 1.9
-	lib:libgpgme${secondaryArchSuffix} = $libgpgmeVersion compat >= 11
-	lib:libgpgmepp${secondaryArchSuffix} = $libgpgmeppVersion compat >= 6
-	lib:libqgpgme${secondaryArchSuffix} = $libqgpgmeVersion compat >= 7
+	gpgme$secondaryArchSuffix = $portVersion compat >= 1.17
+	cmd:gpgme_json$secondaryArchSuffix = $portVersion compat >= 1.17
+	cmd:gpgme_tool$secondaryArchSuffix = $portVersion compat >= 1.17
+	lib:libgpgme$secondaryArchSuffix = $libgpgmeVersionCompat
+	lib:libgpgmepp$secondaryArchSuffix = $libgpgmeppVersionCompat
+	lib:libqgpgme$secondaryArchSuffix = $libqgpgmeVersionCompat
 	"
 REQUIRES="
-	haiku${secondaryArchSuffix}
-	lib:libassuan${secondaryArchSuffix}
+	haiku$secondaryArchSuffix
+	lib:libassuan$secondaryArchSuffix
 	lib:libglib_2.0$secondaryArchSuffix
-	lib:libgpg_error${secondaryArchSuffix}
-	lib:libQt5Core${secondaryArchSuffix}
+	lib:libgpg_error$secondaryArchSuffix
+	lib:libQt5Core$secondaryArchSuffix
 	"
 
 PROVIDES_devel="
-	gpgme${secondaryArchSuffix}_devel = $portVersion compat >= 1.9
-	cmd:gpgme_config${secondaryArchSuffix} = $portVersion compat >= 1.9
-	devel:libgpgme${secondaryArchSuffix} = $libgpgmeVersion compat >= 11
-	devel:libgpgmepp${secondaryArchSuffix} = $libgpgmeppVersion compat >= 6
-	devel:libqgpgme${secondaryArchSuffix} = $libqgpgmeVersion compat >= 7
+	gpgme${secondaryArchSuffix}_devel = $portVersion compat >= 1.17
+	devel:libgpgme$secondaryArchSuffix = $libgpgmeVersionCompat
+	devel:libgpgmepp$secondaryArchSuffix = $libgpgmeppVersionCompat
+	devel:libqgpgme$secondaryArchSuffix = $libqgpgmeVersionCompat
 	"
 REQUIRES_devel="
-	gpgme${secondaryArchSuffix} == $portVersion base
-	devel:libassuan${secondaryArchSuffix}
+	gpgme$secondaryArchSuffix == $portVersion base
+	devel:libassuan$secondaryArchSuffix
 	"
 
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
-	devel:libassuan${secondaryArchSuffix}
+	devel:libassuan$secondaryArchSuffix
 	devel:libglib_2.0$secondaryArchSuffix
-	devel:libgpg_error${secondaryArchSuffix}
-	devel:libQt5Core${secondaryArchSuffix}
+	devel:libgpg_error$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
 	"
 BUILD_PREREQUIRES="
-	cmd:autoconf
-	cmd:automake
-	cmd:awk
-	cmd:bison
-	cmd:gcc${secondaryArchSuffix}
-	cmd:git
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
 	cmd:gpg$secondaryArchSuffix
 	cmd:gpgconf$secondaryArchSuffix
 	cmd:gpg_error$secondaryArchSuffix
-	cmd:grep
-	cmd:kill
```

Comment:
These are used below in the BUILD function, together with `ps` and `xargs` below. Shouldn't these be kept in therefore?

Reply:
It works with or without them in the recipe, can put them back in.

Reply:
Added back in.